services/skus/service_nonint_test.go

```diff
@@ -805,7 +805,7 @@ func TestCreateOrderWithReceipt(t *testing.T) {
 		},
 
 		{
-			name: "successful_case",
+			name: "successful_case_android_leo_monnthly",
```

Comment:
typo?

Reply:
Thanks, fixed in https://github.com/brave-intl/bat-go/pull/2426.